### PR TITLE
Clarified the NERDTreeChangePermissions prompt

### PR DESCRIPTION
**Pull request by rzvxa**
### Description of Changes
Related to #1348     

---
#### Author's Instructions
- [ ] Derive a new `MAJOR.MINOR.PATCH` version number. Increment the:
    - `MAJOR` version when you make incompatible API changes
    - `MINOR` version when you add functionality in a backwards-compatible manner
    - `PATCH` version when you make backwards-compatible bug fixes
- [ ] Update [CHANGELOG.md](https://github.com/scrooloose/nerdtree/blob/master/CHANGELOG.md), following the established pattern.
#### Collaborator's Instructions
- [ ] Review [CHANGELOG.md](https://github.com/scrooloose/nerdtree/blob/master/CHANGELOG.md), suggesting a different version number if necessary.
- [ ] After merging, tag the commit using these (Mac-compatible) bash commands:
    ```bash
    git checkout master
    git pull
    sed -n "$(grep -n -m2 '####' CHANGELOG.md | cut -f1 -d: | sed 'N;s/\n/,/')p" CHANGELOG.md | sed '$d'
    git tag -a $(read -p "Tag Name: " tag;echo $tag) -m"$(git show --quiet --pretty=%s)";git push origin --tags
    ```

### CODE REVIEW

File: nerdtree_plugin/fs_menu.vim [hunk 1]
```diff
@@ -374,7 +374,7 @@ endfunction
 " FUNCTION: NERDTreeChangePermissions() {{{1
 function! NERDTreeChangePermissions()
     let l:node = g:NERDTreeFileNode.GetSelected()
-    let l:prompt = "change node permissions: "
+    let l:prompt = "change node permissions(chmod): "
```
Comment (alerque) [inline nerdtree_plugin/fs_menu.vim:377]:
```suggestion
    let l:prompt = "change node permissions (chmod MODE): "
```

If this needs clarifying, why not be even more explicit like this?

**Review by alerque — CHANGES_REQUESTED** [review]

File: nerdtree_plugin/fs_menu.vim (hunk 1 above)
Comment (rzvxa) [reply nerdtree_plugin/fs_menu.vim:377]: I'm all up for making it clearer, But I'm not sure if chomd MODE can give any more information about the command, I thought of chmod word before the prompt as it mimics the basic schematics of chmod call from the shell.
`change node permissions (chmod): a+x` is really similar to the resulting shell command which is `$ chmod a+x path/to/the/file`.
I don't have any strong objection against either of them and now that I have explained my reasoning I would appreciate it if you make the call about it.

Comment (alerque) [reply nerdtree_plugin/fs_menu.vim:377]: I can see your point, I don't feel super strongly either way. I *do* feel strongly about the need for a space before the opening parenthesis. Fix the space for sure but then pick whatever detail you want. I thought of `(chmod flags)` and `(chmod args)` too. No strong sense of one of them being better than the others. Your call.

Comment (rzvxa) [reply nerdtree_plugin/fs_menu.vim]: I think `chmod args` is a brilliant choice as the user can do something like `u=rw,og=r` as well as `-R +x` so these aren't just sole flags. I'm going to go forward with that one at least for now until someone suggests a better way of prompting this.

**Review by alerque — APPROVED** [review]

**Pull request merged**